project-koku/koku
# adding additional AWS column logic

## Jira Ticket

[COST-####](https://issues.redhat.com/browse/COST-####)

## Description

This change will add additional columns + logic for the customer filtered data flow for AWS

## Testing

1. Checkout Branch
2. Restart Koku
3. Create AWS-local source
4.  Add some data with customer filtered columns (They Like Snake case over there) example `bill_billing_entity`
5. See that the data is ingested correctly

## Notes

...
